### DISCUSSION TITLE
Done by default habits

### DISCUSSION THREAD

**Discussion by Ocaenyth** [iSoron/uhabits#781] [Feature Requests] [answered]
I'm using this app to track many habits, and among which are habits such as `No [X]`, like `No Fish` or `No Meat`, as I am trying to reduce my consumption of these foods.

The issue that I have, is that it is kind of annoying having to remember at the end of every day what you ate, and if it entered those categories. And even once you've gotten the habit it's still cumbersome having to check every single one of them at the end of each day.

While my issue is similar to #96, I think it could be a great feature to have "opposite" habits, where the habit in question will be considered done by default, and you can either have a notification which will "undo" it, or as soon as you've broken the habit, you can uncheck it, and you're good for the day

Example scenario : 
No meat -> Do no eat meat that day
Read -> Read 15 pages of a book

`No meat` would be an "opposite" habit, and `Read` would be a regular one
At the start of the day : 
- `No meat` starts as confirmed/successful
- `Read` starts as not done / not entered (like #96 suggests), and will remain as such until the user sets it as `Done`

Then at noon you decide to have a cheat meal, and eat a burger. Once you've eaten that burger, you can "unset" the `No Meat` habit, meaning that you have "failed" for that day. This way, you don't have to think about remembering it / having to click `No` on your nightly notification, to confirm that you failed this habit
In that case, you could still set up a notification if you want, but instead of marking the habit as `Done`, it would un-mark it (or set it as `not done`)

**Comment by hiqua** [comment, marked as answer]
Duplicate of https://github.com/iSoron/uhabits/discussions/696 (I think)

**Reply by iSoron** [reply]
This is actually closer to #125.

**Reply by Ocaenyth** [reply]
You guys are right, I'll close this issue !

Edit: Actually I'm not sure this is possible, or I couldn't find a way to do it, so I'll mark this reply as answer 
(if it's possible to close and someone can do it, please do)

**Reply by hiqua** [reply]
Great! You can still vote on these features / make suggestions there.

**Reply by Ocaenyth** [reply]
#125 Seems to be closer to what I'm looking for, and from what I've gathered it's currently underway
While #696 is a bit different from what I had in mind, it could also be a solution to my problem (or it seems like a good addition to the app)

I upvoted both requests, but I didn't have anything to add that had not already been said !
